### PR TITLE
Document APICAST_WORKERS param

### PR DESCRIPTION
**Pull request by mayorova**
Additionally, I've reordered the list in alphabetic order for an easier navigation, but can remove the last commit if there are any objections.

### CODE REVIEW

File: doc/parameters.md [hunk 1]
```diff
@@ -194,18 +194,16 @@ It can be used to first load policies from a development directory or to load ex
 
 ### `APICAST_ENVIRONMENT`
 
-**Default**:
-**Value:**: string\[:<string>\]
-**Example**: production:cloud-hosted
+**Value:** string\[:<string>\]
```
Comment (davidor) [inline doc/parameters.md]: I see that you removed the default value. I think that even if it's nil we should document it.

**Comment by davidor** [comment]
Thanks for your contribution @mayorova :+1: 
I think this is clearly an improvement. However, I'm not sure about removing the default value when it's nil. I think it's good to leave it there to be explicit about it. What do you think?

**Comment by mayorova** [comment]
@davidor to me it seems confusing when there is a `Default:` block, but it's empty. Maybe it would be better to say explicitly `Default: <empty>`  or `Default: <none>` in this case.

I am also thinking... Some of the environment variables don't have any default value, but the behavior that the vars are responsible for does have a default in some cases. So, maybe this should be also specified.

**Comment by davidor** [comment]
That's a good point. I think that should be documented. For example, in the case of `APICAST_WORKERS`, by default it's nil, but we should also document what happens when you don't set it, which is that APIcast sets the workers to 1 or `auto` depending on the environment.

I see that in the document there are 2 different strategies for that. In the case of `APICAST_WORKERS` we include a paragraph explaining this. However, in the case of `APICAST_RESPONSE_CODES` we have:  `Default: <empty> (false)`. I'm fine with both approaches.

File: doc/parameters.md (hunk 1 above)
Comment (mayorova) [reply doc/parameters.md]: @davidor I can revert this change, and we can address it separately. Sounds OK?

Comment (davidor) [reply doc/parameters.md]: :+1:

**Comment by mayorova** [comment]
Let's continue the discussion around the default values here: https://github.com/3scale/apicast/issues/652

**Pull request merged**